tests/integration/test_hostgroup.py

```diff
@@ -0,0 +1,189 @@
+from pathlib import Path
+
+import pytest
+import pytest_asyncio
+
+from adcm_aio_client.core.actions._objects import ActionsAccessor
+from adcm_aio_client.core.client import ADCMClient
+from adcm_aio_client.core.errors import MultipleObjectsReturnedError, ObjectDoesNotExistError
+from adcm_aio_client.core.host_groups.action_group import ActionHostGroup, HostsInActionHostGroupNode
+from adcm_aio_client.core.host_groups.config_group import ConfigHostGroup, HostsInConfigHostGroupNode
+from adcm_aio_client.core.objects.cm import Bundle, Cluster, HostProvider
+from tests.integration.bundle import pack_bundle
+from tests.integration.conftest import BUNDLES
+
+pytestmark = [pytest.mark.asyncio]
+
+
+@pytest_asyncio.fixture()
+async def cluster_bundle(adcm_client: ADCMClient, tmp_path: Path) -> Bundle:
+    bundle_path = pack_bundle(from_dir=BUNDLES / "complex_cluster", to=tmp_path)
+    return await adcm_client.bundles.create(source=bundle_path)
+
+
+@pytest_asyncio.fixture()
+async def cluster(adcm_client: ADCMClient, cluster_bundle: Bundle) -> Cluster:
+    return await adcm_client.clusters.create(bundle=cluster_bundle, name="Cluster", description="Cluster description")
+
+
+@pytest_asyncio.fixture()
+async def hostprovider_bundle(adcm_client: ADCMClient, tmp_path: Path) -> Bundle:
+    bundle_path = pack_bundle(from_dir=BUNDLES / "complex_provider", to=tmp_path)
+    return await adcm_client.bundles.create(source=bundle_path)
+
+
+@pytest_asyncio.fixture()
+async def hostprovider(adcm_client: ADCMClient, hostprovider_bundle: Bundle) -> HostProvider:
+    return await adcm_client.hostproviders.create(
+        bundle=hostprovider_bundle, name="Hostprovider name", description="Hostprovider description"
+    )
+
+
+async def test_actionhostgroup(adcm_client: ADCMClient, cluster: Cluster, hostprovider: HostProvider) -> None:
+    for i in range(80):
+        await adcm_client.hosts.create(name=f"test-host-{i}", hostprovider=hostprovider, cluster=cluster)
+
+    for i in range(55):
+        await cluster.action_host_groups.create(
+            name=f"action-host-group-{i}", description=f"action host group description {i}"
+        )
+        action_host_group = await cluster.action_host_groups.get(name__eq=f"action-host-group-{i}")
+        hosts = await cluster.hosts.filter(name__eq=f"test-host-{i}")
+        await action_host_group.hosts.add(host=hosts)
+        await action_host_group.hosts.add(host=await adcm_client.hosts.filter(name__eq=f"test-host-{109 - i}"))
```

Comment:
cryptic number, pls at least add comment about this `109-i`

Reply:
added

Reply:
it hasn't got easier: why this exact host? why 109?

it's easy to understand that we want add something, but other stuff isn't